src/style.css

```diff
@@ -1 +1,49 @@
 /* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+}
+
+ol,
+ul {
+  list-style: none;
+}
```

Comment:
You're styling the `ol` and `ul` tags directly. According to the checklist, you should avoid using tag names for styling (except `html` and `body`). Instead, use classes to style your elements. In this case, you can add a class to your `ol` and `ul` elements and style that class.